src/Aspire.Hosting/Postgres/PostgresBuilderExtensions.cs

```diff
@@ -17,19 +17,19 @@ public static class PostgresBuilderExtensions
     private const string PasswordEnvVarName = "POSTGRES_PASSWORD";
 
     /// <summary>
-    /// Adds a PostgreSQL container to the application model. The default image is "postgres" and the tag is "latest".
+    /// Adds a PostgreSQL resource to the application model. A container is used for local development.
     /// </summary>
     /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
     /// <param name="name">The name of the resource. This name will be used as the connection string name when referenced in a dependency.</param>
-    /// <param name="port">The host port for PostgreSQL.</param>
-    /// <param name="password">The password for the PostgreSQL container. Defaults to a random password.</param>
+    /// <param name="port">The host port used when launching the container. If null a random port will be assigned.</param>
+    /// <param name="password">The administrator password used for the container during local development. If null a random password will be generated.</param>
     /// <returns>A reference to the <see cref="IResourceBuilder{T}"/>.</returns>
-    public static IResourceBuilder<PostgresContainerResource> AddPostgresContainer(this IDistributedApplicationBuilder builder, string name, int? port = null, string? password = null)
+    public static IResourceBuilder<PostgresServerResource> AddPostgres(this IDistributedApplicationBuilder builder, string name, int? port = null, string? password = null)
     {
         password = password ?? Guid.NewGuid().ToString("N");
-        var postgresContainer = new PostgresContainerResource(name, password);
-        return builder.AddResource(postgresContainer)
-                      .WithManifestPublishingCallback(context => WritePostgresContainerResourceToManifest(context, postgresContainer))
+        var postgresServer = new PostgresServerResource(name, password);
+        return builder.AddResource(postgresServer)
+                      .WithManifestPublishingCallback(WritePostgresContainerToManifest)
```

Comment:
(nit on existing code) - the name `WritePostgresContainerToManifest` makes it sound like it is writing the "container" to manifest, and not the resource. `WritePostgresContainerResourceToManifest` is what actually writes the container information. Would it be possible to rename `WritePostgresContainerToManifest` to not have "container" in it?

Reply:
I'll tackle this in a follow up if we want to rename them because all the other resource types are the same so we can refine this naming in one sweep.